File: JavaScriptInDocker/src/figure/Rect.js
import { Figure } from "./Figure.js";

export class Rect extends Figure {
  // private
  _width;
  _height;

  constructor(color, width, height) {
    super(color);
    this._width = width;
    this._height = height;
  }

  getArea() {
    return this._height * this._height;
  }
}

File: Java/src/main/java/com/local/figure/Rect.java
package com.local.figure;

/**
 * Rect
 */
public class Rect extends Figure {
  private double width;
  private double height;

  public Rect(String color, double width, double height) {
    super(color);
    this.width = width;
    this.height = height;
  }

  @Override
  public double getArea() {
    return this.height * this.width;
  }
}
